arrow-libs/core/arrow-core/src/main/kotlin/arrow/core/Either.kt

```diff
@@ -1211,8 +1256,13 @@ inline fun <A, B> Either<A, B?>.leftIfNull(default: () -> A): Either<A, B> =
  * @param elem the element to test.
  * @return `true` if the option has an element that is equal (as determined by `==`) to `elem`, `false` otherwise.
  */
-fun <A, B> Either<A, B>.contains(elem: B): Boolean =
-  exists { it == elem }
+@OptIn(ExperimentalContracts::class)
```

Comment:
Why is this needed?

Reply:
Because the top level one from Either does not apply to extensions :/